src/test/groovy/page/LoginPage.groovy

```diff
@@ -10,5 +10,23 @@ class LoginPage extends Page{
     static at = { title == "Zaloguj się - ExamPlanner" }
 
     static content = {
+        header { $(".u-centered").text() }
+        emailLabel { $(".control-label", 0, for: "Email").text() }
+        emailInput { $("#Email") }
+        passwordLabel { $(".control-label", 0, for: "Password").text() }
+        passwordInput { $("#PasswordPass") }
+        passwordMaskToggle { $("#PasswordSwitch") }
+        rememberPasswordCheckbox { $("#RememberMe", 0, type: "checkbox") }
+        rememberPasswordLabel { $("label", 0, for: "RememberMe").text() }
+        remindPasswordButton { $("input", 0, value: "Przypomnij hasło") }
+        logInButton { $("input", 0, value: "Zaloguj się") }
+    }
+
+    static def "log in with email and password" (email, password) {
```

Comment:
1. Mysle ze static jest tutaj zbedne, pod czas uzycia strony jest tworzony jej obiekt.
2. Lepiej uzywac nazw metod bez nawiasow, na przyklad makeLogin. W nawiasach uzuwamy tylko w testach. (To ulatwi korzystanie z danej metody).
3. Dana metoda jest procedurą, takze warto jej nazwac void zamiast def.